doc/Community-Build/Generate-Cortx-Build-Stack.md

```diff
@@ -0,0 +1,113 @@
+
+# Compile and Build Complete CORTX Stack using Docker
+
+This document provides step-by-step instructions to build and generate the CORTX stack packages using Docker.
+
+To know about various CORTX components, see [CORTX Components guide](https://github.com/Seagate/cortx/blob/main/doc/Components.md).
+
+## Prerequisites
+
+- All the prerequisites specified in the [Building the CORTX Environment for Single Node](Building-CORTX-From-Source-for-SingleNode.md) must be satisfied.
+
+## Procedure
+
+1. Run the following command to clone the CORTX repository:
+
+   ```
+   cd /root && git clone https://github.com/Seagate/cortx --recursive --depth=1
+   ```
+
+2. Run the following command to check out the codebase from the **main** branch for all components:
+
+   ```
+   docker run --rm -v /var/artifacts:/var/artifacts -v /root/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:centos-7.8.2003 make checkout BRANCH=main
```

Comment:
Please remove `-v /var/artifacts:/var/artifacts` from command. It's not required while cloning.

Reply:
@shailesh-vaidya, this will not have any other impact as we had already tested

Reply:
I mean command works but not perfect :smiley: